1.11/installing/ent/custom/configuration/configuration-parameters/index.md

```diff
@@ -89,6 +93,45 @@ This topic provides all available configuration parameters. Except where explici
 | [zk_agent_credentials](#zk-agent)           | ([enterprise type="inline" size="small" /]) The ZooKeeper agent credentials.  |
 
+### adminrouter_tls_1_0_enabled
+
+Indicates whether to enable TLS 1.0 on Admin Router on master nodes. Changing this setting has no effect on internal Admin Router configuration on agent nodes.
+
+- `adminrouter_tls_1_0_enabled: 'true'` Enable the TLS 1.0 protocol on Admin Router.
+- `adminrouter_tls_1_0_enabled: 'false'` Disable the TLS 1.0 protocol on Admin Router. This is the default value.
+
+You are advised not to enable TLS 1.0 as the protocol is considered insecure.
+
+If you have already installed your cluster and would like to change this in-place, you can go through an [upgrade][3] with the `adminrouter_tls_1_0_enabled` parameter set to the desired value.
+
+### adminrouter_tls_1_1_enabled
+
+Indicates whether to enable TLS 1.1 on Admin Router on master nodes. Changing this setting has no effect on internal Admin Router configuration on agent nodes.
+
+- `adminrouter_tls_1_1_enabled: 'true'` Enable the TLS 1.1 protocol on Admin Router master node. This is the default value.
+- `adminrouter_tls_1_1_enabled: 'false'` Disable the TLS 1.1 protocol on Admin Router master node.
+
+If you have already installed your cluster and would like to change this in-place, you can go through an [upgrade][3] with the `adminrouter_tls_1_1_enabled` parameter set to the desired value.
+
+### adminrouter_tls_1_2_enabled
+
+Indicates whether to enable TLS 1.2 on Admin Router on master nodes. Changing this setting has no effect on internal Admin Router configuration on agent nodes.
+
+- `adminrouter_tls_1_2_enabled: 'true'` Enable the TLS 1.2 protocol on Admin Router master node. This is the default value.
+- `adminrouter_tls_1_2_enabled: 'false'` Disable the TLS 1.2 protocol on Admin Router master node.
+
+It is advised to keep this protocol version enabled as its most secure widely supported TLS version.
+
+If you have already installed your cluster and would like to change this in-place, you can go through an [upgrade][3] with the `adminrouter_tls_1_2_enabled` parameter set to the desired value.
+
+### adminrouter_tls_cipher_suite
+
+Provide a custom list of TLS cipher suites. The value will be passed directly into Admin Router's [`ssl_ciphers`](http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_ciphers) configuration directive. There is no validation of this string. Setting it incorrectly will cause DC/OS installation to fail. This configuration settings affects only Admin Router running on DC/OS master nodes.
+
+To validate correctness of provided value use the `openssl ciphers` utility and provide your own value: `openssl ciphers {CUSTOM_VALUE}`. For a list of all available ciphers see the [OpenSSL documentation](https://www.openssl.org/docs/man1.0.2/apps/ciphers.html).
+
+*Note:* Due to Java jurisdiction limitations it is not possible to install DC/OS with only AES256 cipher suites.
```

Comment:
What happens when you try to install DC/OS with only AES256 cipher suites? The answer does not necessarily have to be in the docs, but I'd like to know the answer so that we can agree that it shouldn't.

Nitpick / personal preference - I need to "Note" everything here, so putting "Note" is not useful.

Reply:
This is actually quite important. How should we highlight this?

> What happens when you try to install DC/OS with only AES256 cipher suites?

The installation will fail and you'll never get a healthy DC/OS cluster.

Reply:
Ok, and to be clear we don't have a standard error which we can guarantee?

Reply:
Unfortunately not, the symptom is that `marathon` cannot start due to connection error to the `IAM` service.